src/rpc_client/src/meta_client.rs

```diff
@@ -78,7 +80,14 @@ impl MetaClient {
     }
 
     pub fn worker_id(&self) -> u32 {
-        self.worker_id.expect("worker node id is not set.")
+        self.worker_id.expect("worker node id is set")
+    }
+
+    pub fn host_addr(&self) -> HostAddr {
+        self.host_addr
+            .as_ref()
+            .cloned()
+            .expect("host address is set")
```

Comment:
```suggestion
        self.worker_id.expect("worker node id is not set")
    }

    pub fn host_addr(&self) -> HostAddr {
        self.host_addr
            .as_ref()
            .cloned()
            .expect("host address is not set")
```

Reply:
The recommended message style is "describe the reason you expect the Option should be Some".